driver/src/main/scala/org/mongodb/scala/package.scala

```diff
@@ -358,6 +358,13 @@ package object scala extends ClientSessionImplicits with ObservableImplicits wit
    */
   type ClientEncryptionSettings = com.mongodb.ClientEncryptionSettings
 
+  /**
+   * Alias to Void
+   *
+   * @since 4.0
+   */
+  type Completed = Void
```

Comment:
Where was this type defined previously?

Reply:
It was an Object that used to live here: driver/src/main/scala/org/mongodb/scala/Completed.scala